src/lib/_utilities/i18n.utilities.ts

```diff
@@ -109,7 +109,48 @@ export function interpolate(
     messageDirty = externalText;
   }
 
-  const messageClean = xss(messageDirty);
+  const messageClean = sanitize(messageDirty);
 
   return messageClean;
 }
+
+export function textToKey(text: string | number) {
```

Comment:
This was a wild ride of regex experimentation :)

Reply:
I'm just changing the name of the import, so it makes more sense semantically. Even though the docs use `xss` as the imported function name, I never made sense to me, because it essentially says "cross-site script this value". I prefer "sanitize this value" :)